src/components/threechoicemcq.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+
+class Threechoicemcq extends Component {

```

Comment:
The casing here bugs me a little bit too, why does it have to be a three choice mcq, why not an N choice?

Reply:
cuz it's threechoices

Reply:
It's fine over here.